base_classes/NXoptical_lens.nxdl.xml

```diff
@@ -21,35 +21,36 @@
 #
 # For further information, see http://www.nexusformat.org
 -->
-<!--A draft of a new base class describing an optical lens
+<!--
+A draft of a new base class describing an optical lens
 (Note: NXlens_em describes an electro-magnetic lens or a compound lens)-->
-<definition xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" category="base" name="NXlens_opt" extends="NXcomponent" type="group" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
+<definition xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" category="base" type="group" name="NXoptical_lens" extends="NXcomponent" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
     <symbols>
         <symbol name="N_spectrum">
             <doc>
-                 Size of the wavelength array for which the refractive index of the material
-                 is given.
+                Size of the wavelength array for which the refractive index of the material
+                is given.
             </doc>
         </symbol>
         <symbol name="N_spectrum_coating">
             <doc>
-                 Size of the wavelength array for which the refractive index of the coating
-                 is given.
+                Size of the wavelength array for which the refractive index of the coating
+                is given.
             </doc>
         </symbol>
         <symbol name="N_spectrum_RT">
             <doc>
-                 Size of the wavelength array for which the reflectance or transmission of
-                 the lens is given.
+                Size of the wavelength array for which the reflectance or transmission of
+                the lens is given.
             </doc>
         </symbol>
     </symbols>
     <doc>
-         Description of an optical lens.
+        Description of an optical lens.
```

Comment:
Is this base class for circular lenses only? Best to say so.

Reply:
Should not be constrained, NXcomponent should in the future get shape descriptors added but follow-up proposal